pkgs/by-name/ar/arjun-cnf/package.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, cryptominisat
+, boost
+, louvain-community
+, lib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "arjun-cnf";
+  version = "2.5.2";
+
+  src = fetchFromGitHub {
+    owner = "meelgroup";
+    repo = "arjun";
+    rev = finalAttrs.version;
+    hash = "sha256-5duc05s654HLjbf+dPgyMn6QUVvB0vLji3M4S2o/QYU=";
+  };
+
+  # Can be removed after next release
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/meelgroup/arjun/commit/34188760f1ab4b1b557c45ccaee8d2b9b6f0b901.patch";
+      hash = "sha256-E/yk2ohHP2BAFg353r8EU01bZCqeEjvpJCrBsxPiOWM=";
+    })
+    # Based on https://github.com/meelgroup/arjun/commit/99c4ed4ad820674632c5d9bbcc98c001f8cac98f
```

Comment:
whats is the motivation for not using that patch as is ?

Reply:
Upstream patch has a conflict and cannot be applied